### PR TITLE
Update updating-an-existing-application.md

### PR DESCRIPTION
**Pull request by shahabhijeet**
I was confused that I need to add ExcludeAssets="runtime" to every msbuild package reference.
But in this case, you should add it to every MSBuild package reference except Microsoft.Build.Locator

### CODE REVIEW

**Comment by PRMerger19** [comment]
@shahabhijeet : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
@rainersigwald Rainer, this look OK?

File: docs/msbuild/updating-an-existing-application.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ For example, you can use this XML:
 <ItemGroup>
   <PackageReference Include="Microsoft.Build" Version="15.1.548" ExcludeAssets="runtime" />
   <PackageReference Include="Microsoft.Build.Utilities" Version="15.1.548" ExcludeAssets="runtime" />
+  <PackageReference Include="Microsoft.Build.Locator" Version="1.1.2"  />
```
Comment (rainersigwald) [inline docs/msbuild/updating-an-existing-application.md]: This is specified separately in the `Add package reference` section below. For step-by-step upgrade instructions, I think it makes sense to do the upgrade in two phases:

* Move to package references
* Move to Locator

But this is definitely confusing if you just come to this document looking for "what does this look like in the end?" which is a totally reasonable thing to do. @Mikejo5000 should we have a separate doc for that use case and establish some cross-links?

Comment (ghogen) [reply docs/msbuild/updating-an-existing-application.md]: @Mikejo5000 can you take a look here and recommend the next step?

Comment (Mikejo5000) [reply docs/msbuild/updating-an-existing-application.md]: @rainersigwald Rainer, the new non-sequential use case here would be "How to reference MSbuild"? Which would apply equally to MSBuild 15.0 and 16.0.

**Comment by Mikejo5000** [comment]
Opened a PR to make a few minor clarifications

**Pull request closed without merging**